public/js/AppSorting.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable no-undef */
+
+const sortingAlpha = (symbols, direction) =>
+  _.orderBy(
+    symbols,
+    s => {
+      return s.symbol;
+    },
+    direction
+  );
+
+const sortingSellProfit = (symbols, direction) =>
+  _.orderBy(
+    symbols,
+    s => {
+      return s.sell.currentProfit;
+    },
+    direction
+  );
+
+const sortingBuyDifference = (symbols, direction) =>
+  _.orderBy(
+    symbols,
+    s => {
+      return s.buy.difference;
```

Comment:
I am not sure here, but I think this is also treated as string, we can put a `+` just to be safe.

Reply:
Updated.